docs/cn/build_and_run.md

```diff
@@ -20,12 +20,17 @@ mk-tar.sh 和 mk-deb.sh 用于 curve v2.0 之前版本的编译打包，v2.0 版
 docker pull opencurvedocker/curve-base:build-debian9
 ```
 
+debian9 一些包已经过时, 建议使用 debian10
```

Comment:
This [pr2165](https://github.com/opencurve/curve/pull/2165) removes support for debian10 and debian11, the mirrors of these two are not in the official ci and thus lack many software.
It is recommended to upgrade the image according to [pr2165] and the current image before prompting to use debian10, otherwise debian9 is more stable.
  [pr2165]: https://github.com/opencurve/curve/pull/2165

Reply:
> This [pr2165](https://github.com/opencurve/curve/pull/2165) removes support for debian10 and debian11, the mirrors of these two are not in the official ci and thus lack many software. It is recommended to upgrade the image according to [pr2165] and the current image before prompting to use debian10/11, otherwise debian9 is more stable. [pr2165]: #2165

ok, some friends said this. Think debian9 is too old. Stable is better.

Reply:
> > This [pr2165](https://github.com/opencurve/curve/pull/2165) removes support for debian10 and debian11, the mirrors of these two are not in the official ci and thus lack many software. It is recommended to upgrade the image according to [pr2165] and the current image before prompting to use debian10/11, otherwise debian9 is more stable. [pr2165]: #2165
> 
> ok, some friends said this. Think debian9 is too old. Stable is better.

I think so too, but replacing the base image requires a stability test, and the limit is not high. Raise an issue and record it?